### ISSUE TITLE
eval and test dataset

### ISSUE THREAD

**Issue by bino282** [STAIR-Lab-CIT/STAIR-captions#1]
I downloaded the data but only see the val file but not the test file.

**Comment by LiJiaBei-7** [comment]
113,287 images for the training set, 5,000 images for the validation set, and 5,000 images for the test set.
It should be necessary to re-divide the train and val
